src/dotnet/APIView/APIViewWeb/Pages/Shared/_Layout.cshtml

```diff
@@ -60,6 +60,8 @@
         <script src="~/lib/jquery/dist/jquery.sticky-sidebar.js"></script>
         <script src="~/lib/ResizeSensor/dist/ResizeSensor.js"></script>
         <script src="~/lib/bootstrap/dist/js/bootstrap.bundle.js"></script>
+        <script src="~/lib/mark/dist/jquery.mark.js"></script>
```

Comment:
Why do we need this?

Reply:
this is the dist for dev, just like we have for the other libraries.

Reply:
Which I forgot to commit..